packages/reacord/library/internal/renderers/renderer.ts

```diff
@@ -51,7 +51,7 @@ export abstract class Renderer {
 
     setTimeout(() => {
       this.updates.next({ action: "deferUpdate", interaction })
-    }, 500)
+    }, 1500)
```

Comment:
When responding to interactions, sometimes defer update fails with being called already, this slightly fixes that although it still encounters errors, just less frequently

Reply:
This timeout might just be a flawed way to handle the issue 🤔 I can't think of anything better at the moment though